spec/game_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/game'
+
+describe Game do
+  describe '#add' do
+    subject(:game) { described_class.new }
+
+    it 'adds the coin to the bottom of the column' do
+      column = 3
+      game.add(column)
+      symbol = 'X'
+      expected = game.board.positions[5][column]
+      expect(expected).to eql symbol
+    end
+    it 'adds the second coin on top of last one' do
+      column = 3
+      game.add(column)
+      game.add(column)
+      symbol = 'X'
+      expected = game.board.positions[4][column]
+      expect(expected).to eql symbol
+    end
+    it "doesn't do anything if the column is full" do
+      positions = Array.new(6) { Array.new([nil, nil, nil, 'X', nil, nil, nil]) }
+      column = 3
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      expected = game.board.positions
+      expect(expected).to eql positions
+    end
+  end
+
+  describe '#change_player' do
+    subject(:game) { described_class.new }
+    it 'changes active player' do
+      player = 'Player 2'
+      game.change_player
+      expect(game.active_player).to eql player
+    end
+  end
+
+  describe '#column_full?' do
+    subject(:game) { described_class.new }
+
+    before do
+      column = 3
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+      game.add(column)
+    end
+
+    it 'returns true when column is full' do
+      expect(game.column_full?(3)).to be true
+    end
+
+    it 'returns false if column is not full' do
+      expect(game.column_full?(5)).to be false
+    end
+  end
+
+  describe '#win?' do
```

Comment:
In cases like this, where you have a complex logic, it's important to make sure your whole business logic is covered with tests. You could use https://github.com/simplecov-ruby/simplecov to check your code coverage.

Specifically, you need tests to cover each case that can lead to a win, diagonal, horizontal etc.